### DISCUSSION TITLE
Making account public!

### DISCUSSION THREAD

**Discussion by ghost** [community/community#141955] [Repositories]
How to make may repositories public?

**Comment by yolorabbit777** [comment]
Hi @Banti67 ,

Hope you are doing well.

Please follow these steps to make your repository public.
- Go to your repository url
Like https://github.com/yolorabbit/yolorabbit
- Go to Settings tab
When you click Settings, you can scroll down to the Danger Zones.
![image](https://github.com/user-attachments/assets/73e44114-b4cd-4137-957f-b6a62221a8b0)
You can click Change Visibility and make it public.

I wish it helps you.

Best,
@yolorabbit
